src/main/res/xml/app_settings.xml

```diff
@@ -119,6 +119,13 @@
         android:title="@string/category_about">
 
         <Preference
+            android:title="@string/open_log"
+            android:summary="@string/open_log_summary">
+            <intent
+                android:action=".activities.LogActivity" />
```

Comment:
Needs to be this:
```
            <intent
                android:action="android.intent.action.VIEW"
                android:targetPackage="com.nutomic.syncthingandroid"
                android:targetClass="com.nutomic.syncthingandroid.activities.LogActivity"
                android:data="open_source_licenses" />
```

Unfortunately, this doesn't work in debug builds, because we change the package name there. So the best solution is probably to send the intent from Java.